jax/lax_reference.py

```diff
@@ -12,92 +12,175 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# type: ignore
```

Comment:
@mattjj This was necessary to satisfy down mypy. It's unable to find the type of `lax.neg_p` in `np_impl[lax.neg_p] = neg` etc. Have you had this problem before?

Reply:
In general you may have to tweak things a little to make mypy happy. I'm not sure what specifically is going wrong here.

Reply:
I already tweaked for a while, with no results. Let me know if you have further ideas!

Reply:
No longer a problem since `np_impl` is now populated directly in `numpy_eval.py`.